compiler/rustc_middle/src/mir/terminator.rs

```diff
@@ -527,7 +527,8 @@ impl<'tcx> TerminatorKind<'tcx> {
                         InlineAsmOperand::InOut { reg, late, in_value, out_place: None } => {
                             write!(fmt, "in{}out({}) {:?} => _", print_late(late), reg, in_value)?;
                         }
-                        InlineAsmOperand::Const { value } => {
+                        InlineAsmOperand::Const { value, .. } => {
+                            // FIXME consider span?
```

Comment:
@oli-obk what should we do here? I think I wouldn't just do `write!(fmt, "span {:?}", span)` but unsure ...

Reply:
hmm, yea, just print the span separately as you described